packaging/packaging.go

```diff
@@ -175,14 +183,28 @@ func (p *FilePackager) writeTarball(tarFileName, manifestFileName string, archiv
 	defer tw.Close()
 
 	// add the files to the tarFile
+	var files []string
+	files = append(files, manifestFileName)
 	for idx, filePath := range archiveFiles {
 		if strings.HasSuffix(filePath, ".csv") {
 			uploadName := p.uid + "_openshift_usage_report." + strconv.Itoa(idx) + ".csv"
+			files = append(files, uploadName)
 			if err := p.addFileToTarWriter(uploadName, filePath, tw); err != nil {
 				return fmt.Errorf("writeTarball: failed to create tar file: %v", err)
 			}
+		} else {
+			files = append(files, filePath)
```

Comment:
Why is this `else` here?

Reply:
It is pre-emptive because i am going to add a file to the payload in a future PR with the CR status/any errors and it will be in json format

Reply:
🤔  Why would we add a new file instead of just adding it into a field in the manifest file?

Reply:
I'll remove it for now, but the story is left undefined on if we want to add to the manifest or add a new file so I might add it back when I pick up that issue (just depends on the what makes the most sense)